specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -4,149 +4,85 @@ This specification refers to the following
 documents. The terms used to refer to the documents in this
 specification are included in parentheses.
 
- _Java™ Platform, Enterprise Edition
-Specification Version 8_ (this specification). Available at
-_http://jcp.org/en/jsr/detail?id=366_ .
+Jakarta™ EE Platform Specification Version 8. Available at: <link needed>.
```

Comment:
Following the link in @bshannon's comment, the missing ink should be along the lines of http://jakarta.ee/specifications/full-platform/8.0.1.  The tricky part will be to figure out the exact version that will be used.  But, take your best guess for now.